spring-cloud-dataflow-server-core/src/test/java/org/springframework/cloud/dataflow/server/controller/JobInstanceControllerTests.java

```diff
@@ -169,7 +171,9 @@ private void createSampleJob(String jobName, int jobExecutionCount)
 		TaskExecution taskExecution = taskExecutionDao.createTaskExecution(jobName, LocalDateTime.now(), new ArrayList<String>(), null);
 
 		for (int i = 0; i < jobExecutionCount; i++) {
-			JobExecution jobExecution = jobRepository.createJobExecution(jobName, new JobParameters());
+			JobParameters jobParameters =
+				new JobParameters(Collections.singletonMap("parm", new JobParameter<>(i, Integer.class)));
```

Comment:
Is `parm` the correct key?

Reply:
Great question.    In this case we used to be able to create multiple job executions with a single instance for this command.   But that loophole was closed in Batch 5.  So now we need to make each JobExecution unique by either changing the name of the Job or adding a unique parameter value.   In our case we want the same job name, so we can add a unique job parameter.

Reply:
Oh ok, so this is in effect a "dummy param", correct? If so, maybe we should name the key something besides `parm` - maybe `unused` (or something like that). Wdyt?

Where does this show up as `parm`? In the UI?

Reply:
Job Parameters can be viewed as a part of the Job Instance in the UI.   I can change it to test-parameter.

Reply:
Oh nevermind -  This is a test. Carry on.